js/main.js

```diff
@@ -93,7 +130,20 @@ function parse_code(code) {
 		payload = b64(parts[1]);
```

Comment:
Should this switch to jws.something?

Reply:
I tried to figure out how to use `jws.payloadPP()` here, but couldn't find a sane (or any, really) way to convert its output into the `UInt8Array` correctly such that pako can use it. I'm not sure if that's because `jws.payloadPP` is doing something incorrect or what, but it didn't make much sense to me that it is a 'binary' `string`. I'm not really a JS guy though, this might be a normal pattern for this and I just don't know how to use it.

Reply:
No worries. I always hack my javascript as well, so I'm not sure :)  I did start this project with jws, but I couldn't make it work, but I don't recall why.

Mind adding a quick:

 // TODO Switch this to jws.payloadPP if we can get uncompress.

So the next person doesn't have the same discussion as us :)